scripts/train_cnn.ipynb

```diff
@@ -211,8 +211,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "idx_not_download = df_download.set_index('image_name').drop(downloaded)['row'].values.tolist()\n",
-    "df_download.drop(idx_not_download, inplace=True)"
+    "downloaded =  list(set(downloaded).intersection(set(df_download['image_name'])))\n",
```

Comment:
why not do `df_download = df_download.set_index('image_name')[downloaded].reset_index()`